.github/actions/link-check/entrypoint.sh

```diff
@@ -5,7 +5,7 @@ MAX_WAIT_TIME=600 # 5 min
 
 # Provide default options if none are provided
 HUGO_OPTIONS="${HUGO_OPTIONS:=-b http://localhost/developer -p 1314}"
-MUFFET_OPTIONS="${MUFFET_OPTIONS:=-t 30 -e 'https?' -e 'https?:\/\/localhost.*' --exclude='/developer/get-workshop'}"
+MUFFET_OPTIONS="${MUFFET_OPTIONS:=-t 30 -e 'https?' -e 'https?:\/\/localhost.*' -e '/developer/get-workshop'}"
```

Comment:
I think `-e 'https?' -e 'https?:\/\/localhost.*'` is redundant but I left it alone!